src/component/cyfs-core/protos/core_objects.proto

```diff
@@ -10,6 +10,49 @@ message StorageBodyContent {
     bytes value = 1;
 }
 
+// ObjectShellObject
+message ObjectShellDescContent {
```

Comment:
It's about: https://github.com/buckyos/CYFS/issues/219

Reply:
I'm sorry, I have commit it with the feature #71 .